osltweaks.php

```diff
@@ -16,6 +16,35 @@ function osltweaks_civicrm_buildForm($formName, &$form) {
       // include some javascript to finagle the wording in that section.
       CRM_Core_Resources::singleton()->addScriptFile('org.osltoday.osltweaks', 'js/CRM_Contribute_Form_Contribution_Main-isShowCMS.js');
     }
+
+    // Array of Contribution page id for the Country and State rewrite
+    $settings = CRM_Core_BAO_Setting::getItem(NULL, 'com.joineryhq.osltweaks');
+
+    foreach ($settings['page_ids'] as $key => $id) {
+      // Check if contribution page id is match on the $settings and make changes
+      if ($id == $_GET['id']) {
```

Comment:
Instead of `$_GET['id']`, please use `$form->_id` to get the `id` of the page. If we use `_GET`, then this will only work on pages where `id` is in the URL, but really we don't know if sometimes that might be missing, because CiviCRM may have some other URLs where it displays a contribution page. But `$form->_id` will always be there, because it's added into the $form object by civicrm core code. So `$form->_id` is more robust and less likely to break.

Reply:
Done